config/app.config

```diff
@@ -27,8 +27,9 @@ whitelisted_warnings: ["ANS1809W", "ANS2042W", "ANS2250W"]
 
 # Exclude filters (different on biotank and Irma)
 #
-# Dirs to exclude from the _archive copy of the runfolder/project
+# Dirs and file extensions to exclude from the _archive copy of the runfolder/project
 exclude_dirs: ["Data", "Thumbnail_Images"]
+exclude_extensions: [".bcl"]
```

Comment:
While at it, maybe you can add `.cbcl` too?

Reply:
Well, this config file is only a template. On production systems we use copies of this config file that are under /etc somewhere, so adding .cbcl here wouldn't actually change anything. And since we exclude the whole Data dir we should be fine. If you're ok with this let's go ahead and merge!